src/main/java/uk/gov/digital/ho/hocs/info/application/CacheScheduler.java

```diff
@@ -1,23 +0,0 @@
-package uk.gov.digital.ho.hocs.info.application;
-
-import lombok.AllArgsConstructor;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Profile;
-import org.springframework.scheduling.annotation.EnableScheduling;
-import org.springframework.scheduling.annotation.Scheduled;
-import uk.gov.digital.ho.hocs.info.api.TeamService;
-import uk.gov.digital.ho.hocs.info.api.UserService;
-
-@Configuration
-@EnableScheduling
-@AllArgsConstructor
-@Profile("!test")
-public class CacheScheduler {
-
-    private UserService userService;
-
-    @Scheduled(fixedDelayString = "${cache.user.refresh}000")
```

Comment:
`cache.user.refresh` can get rid of this from the application.properties

Reply:
I have made this change